### DISCUSSION TITLE
麻烦问一下，nas上的calibre-web能用这个吗？不知道该怎么选择版本

### DISCUSSION THREAD

**Discussion by wk-coder** [Cirn09/calibre-do-not-translate-my-path#19] [General]
请教一下，万分感谢

**Comment by Cirn09** [comment]
简单看了一下， calibre-web 似乎没有复用 calibre 代码，是个单纯解析 calibre 书库的方法

如果我的分析没问题，那么 calibre-web 兼容本项目书库，不用做什么修改

你先试试，有问题再说

**Comment by wk-coder** [comment]
兼容的意思是不需要使用这个项目吗？ 暂时就是自己导入进去几本书，试了下没有问题，不知道别人说的拼音是指哪一步出的问题

**Comment by chinaeaa** [comment]
@wk-coder 
calibre自建中文书库，加入中文电子书时会用拼音去建电子书目录，非常讨厌，看拼音很难知道目录名到底是什么意思，并且calibre开发者拒绝为其它语言进行目录存储更改。
一直在用calibre-web，使用了楼主的版本建立了书库，很好用没什么问题。

**Comment by wk-coder** [comment]
@chinaeaa  暂时没有路径问题了，docker鼓捣了半天，一开始用的官方的镜像，弄好了豆瓣识别，结果不能转换格式。后来换了另一个
johngong/calibre-web，可以转换格式，而且目录也都是中文，就是豆瓣不能用，试了好多次没有成功
